src/includes/dockerfile/Pgpool-3.3-3.7.part.Dockerfile

```diff
@@ -27,15 +27,21 @@ RUN  apt-get install -y postgresql-client-{{ PG_CLIENT_VERSION }}
 {{ /PG_CLIENT_LATEST }}{{ ^PG_CLIENT_LATEST }}
 RUN TEMP_DEB="$(mktemp)" && \
     wget -O "$TEMP_DEB"  "http://atalia.postgresql.org/morgue/p/postgresql-{{ PG_CLIENT_VERSION }}/postgresql-client-{{ PG_CLIENT_VERSION }}_{{ PG_CLIENT_PACKAGE_VERSION }}_amd64.deb" && \
-    (dpkg -i "$TEMP_DEB" || apt-get install -y -f) && rm -f "$TEMP_DEB"
+    dpkg -i "$TEMP_DEB" && rm -f "$TEMP_DEB"
 {{ /PG_CLIENT_LATEST }}
 
 {{ #PGPOOL_LATEST }}
 RUN  apt-get install -y libpgpool0={{ PGPOOL_VERSION }}\* pgpool2={{ PGPOOL_VERSION }}\*
 {{ /PGPOOL_LATEST }}{{ ^PGPOOL_LATEST }}
+RUN  apt-get install -y postgresql-common libmemcached11
```

Comment:
That is exactly why you need `|| apt-get install -y -f` which you removed
So you don't need install dependencies manually

Reply:
See detailed logs in issue: https://github.com/paunin/PostDock/issues/186

Reply:
I don't think we must follow community bugs in this case :) We need controlled way to install what we want.